<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyPresto</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="leftSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="tablesWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>800</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed</set>
        </property>
        <property name="indentation">
         <number>15</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>DBs/Tables</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="schemaView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="indentation">
         <number>15</number>
        </property>
        <column>
         <property name="text">
          <string>Field</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="dataLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="connWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="connWidgetLayout">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="urlLabel">
             <property name="text">
              <string>Url:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="urlEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="userLabel">
             <property name="text">
              <string>Presto User:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="userEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tunnelButton">
             <property name="text">
              <string>Tunnel</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDialogButtonBox" name="connButtonBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="standardButtons">
              <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="tunnelWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#tunnelWidget {background-color: lightgray}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="serverLabel">
             <property name="text">
              <string>PrestoServer:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="serverEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gatewayLabel">
             <property name="text">
              <string>Gateway:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="gatewayEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tunnelUserLabel">
             <property name="text">
              <string>User:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tunnelUserEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>130</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="keyComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>110</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="currentText">
              <string>KeyFile</string>
             </property>
             <item>
              <property name="text">
               <string>KeyFile</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Password</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pwdLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>188</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>188</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Enter Password...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="keyFileButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Choose Key File...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="TextEdit" name="sqlEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Write your Presto SQL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="sqlWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="sqlWidgetLayout">
           <item>
            <widget class="QDialogButtonBox" name="runButtonBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="standardButtons">
              <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="dataWidget">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
